splice_si_api/src/main/java/com/splicemachine/si/impl/store/IgnoreTxnSupplierImpl.java

```diff
@@ -105,4 +141,18 @@ private void init() throws IOException {
             }
         }
     }
+
+    @Override
+    public void refresh() {
+        synchronized (this) {
+            cache.clear();
+            try {
+                populateIgnoreTxnCache();
+            } catch (IOException e) {
+                LOG.error("Couldn't populate Ignore Cache", e);
+                // Force restart
+                // System.exit(-1);
```

Comment:
Remove comments

In any case, if this fails we need something more than just logging an error, can we fail the original ROLLBACK_TO_TXN call?

Reply:
Was this question referring to the call to refresh in SpliceDefaultCompactor?  I've since removed that call based on the previous comment.  The only other call to refresh is in the LEAVE_RESTORE_MODE block of SpliceDatabase.registerDDL() which would occur late in the ROLLBACK_TO_TXN process.  Should ROLLBACK_TO_TXN still be failed at this point, as it's basically in cleanup phase with the main work already done?

Reply:
No, it was a general comment, but you are right it's late in the execution already. I'm not sure how to handle a failure here to be honest, maybe we should really restart here, otherwise this could lead to inconsistencies if it's not populated correctly. Do you have an opinion? Otherwise we can open a JIRA to discuss it later.